kafka/adminapi.go

```diff
@@ -560,6 +570,10 @@ type DeleteACLsResult = DescribeACLsResult
 // The returned result event is checked for errors its error is returned if set.
 func (a *AdminClient) waitResult(ctx context.Context, cQueue *C.rd_kafka_queue_t, cEventType C.rd_kafka_event_type_t) (rkev *C.rd_kafka_event_t, err error) {
 
+	if a.verifyAdminClient() == false {
```

Comment:
Maybe we need this check only for the public methods and not the private ones (since we can control when the private ones are called and take care never to call one after closure.) Public methods on the other hand are called by the external users.